docs/reference/sdks.mdx

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 2
+id: sdks
+title: SDKs
+---
+
+# SDKs
+
+SDKs for the evaluation of feature flags and the development of OpenFeature compatible components.
+
+## Server SDKs
+
+import { SDKComponentCard } from '@site/src/components/custom';
```

Comment:
It doesn't look like this import is used

Reply:
I think you're right!

Reply:
Fixed. Unfortunately the .mdx extension doesn't have very advanced features - it can't seem to identify unused imports.